browser/brave_profile_prefs.cc

```diff
@@ -335,6 +335,13 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   brave_ads::RegisterP2APrefs(registry);
 #endif
 
+#if !defined(OS_ANDROID)
+  // Turn on most visited mode on NTP by default.
+  // We can turn customization mode on when we have add-shortcut feature.
+  registry->SetDefaultPrefValue(prefs::kNtpUseMostVisitedTiles,
```

Comment:
This value might be getting overridden by Chromium (I'm not 100% sure)

Basically in https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/containers/newTab/gridSites.tsx#L78, the tile is becoming draggable. If you drag this tile, it does actually move it and then it seems to activate custom links mode

Reply:
After troubleshooting, I re-ran `npm run init` for this branch and built... and it worked perfectly 😄 I believe the problem was on my side. Everything looks great!